dependencies.props

```diff
@@ -9,6 +9,7 @@
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
     <CoreFxExpectedPrerelease>beta-24911-02</CoreFxExpectedPrerelease>
```

Comment:
Do we need to do anything special to get this repo *subscribing* to core-setup updates now that it is consuming `Microsoft.NETCore.App`?  /cc @dagood 

Alternatively we could have this consume the private CoreFx package and CoreClr package instead.  /cc @weshaggard